dap-launch.el

```diff
@@ -0,0 +1,241 @@
+;; dap-mode-launch-json.el --- support launch.json -*- lexical-binding: t -*-
+
+;; Copyright (C) 2020 Nikita Bloshchanevich
+
+;; Author: Nikita Bloshchanevich <nikblos@outlook.com>
+;; Keywords: languages
```

Comment:
as this file is part of the `dap-mode` package, I think ẃe don't need these 2 lines
```suggestion
```